stacker/util.py

```diff
@@ -296,17 +298,40 @@ def yaml_to_ordered_dict(stream, loader=yaml.SafeLoader):
         OrderedDict: Parsed YAML.
     """
     class OrderedLoader(loader):
-        """Subclass of Python yaml class."""
-        pass
+        """Subclass of the given Python loader yaml class."""
```

Comment:
I think the name of this should probably be changed - maybe `OrderedUniqueLoader` or something?  Also, probably worth describing what it does in the docstring rather than this generic one.

Reply:
fixed